google/cloud/aiplatform/metadata/metadata.py

```diff
@@ -47,7 +47,7 @@ def set_experiment(self, experiment: str):
         )
         self._experiment = context
 
-    def set_run(self, run: str):
+    def start_run(self, run: str):
```

Comment:
might need to check the ValueError descriptions, I'm not sure if we were mentioning to call set_run somewhere?

Reply:
Thanks for catching this. I updated all the ValueErrors and also some comments in the file to start_run